indico/modules/designer/controllers.py

```diff
@@ -339,6 +339,11 @@ def _process_GET(self):
     def _process_POST(self):
         data = dict({'background_position': 'stretch', 'items': []}, **request.json['template'])
         self.validate_json(TEMPLATE_DATA_JSON_SCHEMA, data)
+
+        if backside_template_id := request.json.get('backside_template_id'):
+            if backside := DesignerTemplate.get(backside_template_id):
```

Comment:
It's nicer to do this via webargs/marshmallow ;)

```py
@use_kwargs({
    'backside': ModelField(DesignerTemplate, data_key='backside_template_id'),
})
```

That way you also avoid ugly DB errors when sending crap that's not a number in the json field

Reply:
I didn't know about this, nice!